get_last_revision_titles.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+"""
+    get_last_revision.py
+
+    MediaWiki Action API Code Samples
+    Demo of `Revisions` module: Get data including content of last revision of titles [[API]] and [[Main Page]]
```

Comment:
This line could be rephrased to "Get revision data with content for pages with titles ... and ..."